### PR TITLE
cleanup(XCCDF): RHICOMPL-3951 do not save missing RHEL6 benchmarks

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: test/services/xccdf_report_parser_test.rb [hunk 1]
```diff
@@ -55,22 +55,6 @@ def package_name
       end
     end
 
-    should 'not save missing benchmarks for RHEL8' do
```
Comment (romanblanco) [inline test/services/xccdf_report_parser_test.rb:58]: @skateman Should this test also be removed?

Comment (skateman) [reply test/services/xccdf_report_parser_test.rb:58]: Yes, as the saving code is deleted, so we cannot really test its opposite.

Comment (romanblanco) [reply test/services/xccdf_report_parser_test.rb:58]: Confirmed, that the saving logic is intended to be removed.

**Review by romanblanco — APPROVED** [review]
LGTM :+1:

**Pull request merged**